main/exercise/question.class.php

```diff
@@ -1911,6 +2057,53 @@ public static function displayTypeMenu($objExercise)
         echo '</div>';
     }
 
+    /**
+     * Displays the menu of question types.
+     *
+     * @param Exercise $objExercise
+     */
+    public static function displayPtTypeMenu($objExercise)
+    {
+        // 1. by default we show all the question types
+        $questionTypeList = self::getQuestionPtTypeList();
+
+        echo '<div class="panel panel-default">';
+        echo '<div class="panel-body">';
+        echo '<ul class="question_menu">';
+        foreach ($questionTypeList as $i => $type) {
+            /** @var Question $type */
+            $type = new $type[1]();
+            $img = $type->getTypePicture();
+            $explanation = get_lang($type->getExplanation());
+            echo '<li>';
+            echo '<div class="icon-image">';
+            $icon = '<a href="ptest_admin.php?exerciseId='.$objExercise->id.'&'.api_get_cidreq().'&newQuestion=yes&answerType='.$i.'">'.
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line exceeds maximum limit of 120 characters; contains 136 characters](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5018478)